test/library/standard/BigInteger/apiTest.chpl

```diff
@@ -0,0 +1,222 @@
+use BigInteger;
+use Reflection;
+
+config const executeLocale = 0;
+
+const a = new bigint("123456789012345678901234567890");
+const b = new bigint("123456789012345678901234567891");
+const c = new bigint("246913578024691357802469135780");
+const d = new bigint("-246913578024691357802469135780");
+
+on Locales[min(Locales.domain.high, executeLocale)] {
+  assert((a+1) == b);
+  assert((a+(1:bigint)) == b, getLineNumber());
```

Comment:
Why are we including the line number here?

Reply:
If we don't need it, I think that's the only use of the Reflection module

Reply:
Whoops, leftover from an earlier version.